docs/collections/graphs/quickstart.md

```diff
@@ -22,100 +22,65 @@ Edges have a direction, with their relations `_from` and `_to` pointing from one
 - INBOUND: `_from` ← `_to`
 - ANY: `_from` ↔ `_to`.
 
-## Example
+## Prerequisites
 
-For this example, assume the following credentials:
+- Access to a [Macrometa account](https://auth.paas.macrometa.io/) with sufficient permissions to create streams.
+- Have your API Key ready. For more information, refer to [Create API Keys](../account-management/api-keys/create-api-keys.md).
+- Install the appropriate SDK. For more information, refer to [Install SDKs](../sdks/install-sdks.md).
 
-- Tenant name is `nemo@nautilus.com`.
-- User password is `xxxxxx`.
+### Step 1. Connect to GDN
 
-### SDK Download
+The first step in using the Macrometa Global Data Network (GDN) is to establish a connection to a local region. When this code executes, it initializes the server connection to the region URL you specified. For more information about connecting to the GDN, refer to [Authentication](../account-management/auth/index.md).
```

Comment:
```suggestion
To use Macrometa Global Data Network (GDN), you must first establish connection to a local region. When this code runs, it initializes the server connection to the region URL you specified. For more information about connecting to GDN, refer to [Authentication](../account-management/auth/index.md).
```

@diana-macrometa Have we had a discussion about referring to GDN as "GDN" vs "the GDN"?

Reply:
Not really. I would think it depends on context.